setup.py

```diff
@@ -58,12 +58,14 @@
         'Framework :: Django :: 3',
         'Framework :: Django :: 3.2',
         'Framework :: Django :: 4',
-        'Framework :: Django :: 4.0',
         'Framework :: Django :: 4.1',
+        'Framework :: Django :: 4.2',
         'Framework :: Wagtail',
         'Framework :: Wagtail :: 4',
         'Framework :: Wagtail :: 4.1',
         'Framework :: Wagtail :: 4.2',
+        'Framework :: Wagtail :: 5',
+        'Framework :: Wagtail :: 5.0',
```

Comment:
**Comment** not sure if you want to consider adjusting this. It's something I have had mentioned to me recently.

e.g. https://github.com/torchbox/wagtailmedia/blob/259ac0247fade78a6d447bac968b8881053c7c5c/pyproject.toml#L21

Reply:
Removed specific Wagtail 4.x and 5.x versions